SETUP/tests/unittests/ApiTest.php

```diff
@@ -680,17 +691,125 @@ public function test_validate_text()
         $response = $this->validate_text($project->projectid, "This is an invĀlid test file");
         $expected = [
             'valid' => false,
-            'mark_array' => [["This is an inv", 0], ["Ā", 1], ["lid test file", 0]],
+            'mark_array' => [["This is an inv", "", 0], ["Ā", "LATIN CAPITAL LETTER A WITH MACRON", 1], ["lid test file", "", 0]],
         ];
         $this->assertEquals($expected, $response);
 
         $response = $this->validate_text($project->projectid, "This is a valid test file");
         $expected = [
             'valid' => true,
-            'mark_array' => [["This is a valid test file", 0]],
+            'mark_array' => [["This is a valid test file", "", 0]],
+        ];
+        $this->assertEquals($expected, $response);
+    }
+
+    public function test_wordcheck_world_bad_word()
+    {
+        $project = $this->_create_available_project();
+        $response = $this->wordcheck($project->projectid, "Fort Snelling test file");
+        $expected = [
+            'wc_array' => [["Fort ", 0], ["Snelling", 1], [" test file", 0]],
+            'scripts' => [],
+        ];
+        $this->assertEquals($expected, $response);
+    }
+    /*
+    public function test_wordcheck_site_bad_word()
```

Comment:
This works on the test server but I don't know how to make it work in the Github ci. Perhaps not worth the effort.